feature_layers/feature-layer-definition-expression/README.md

```diff
@@ -1,24 +1,33 @@
-# Feature Layer Definition Expression
+# Feature layer definition expression
 
-Filter which features are shown using an expression.
+Limit the features displayed on a map with a definition expression.
 
-![](FeatureLayerDefinitionExpression.gif)
+![Image of feature layer definition expression](FeatureLayerDefinitionExpression.png)
+
+## Use case
+
+Set a definition expression to filter out the features to be displayed. You might filter a dataset of tree quality selecting for only those trees which require maintenance or are damaged.
 
 ## How to use the sample
 
-Use the buttons in the control panel to apply or reset definition expression.
+Press the 'expression' button to limit the features requested from the feature layer to those specified by the SQL query definition expression. Click the 'expression' button again to de-press it and remove the definition expression on the feature layer, which returns all the records.
 
 ## How it works
 
-To limit the `Feature`s in your `FeatureLayer`:
-
-1. Create a `ServiceFeatureTable` from a URL.
+1. Create a service feature table from a URL.
 2. Create a feature layer from the service feature table.
-3. Set the limit of the features on your feature layer using `FeatureLayer.setDefinitionExpression(Expression)`.
+3. Set the limit of the features on your feature layer using the `DefinitionExpression`.
```

Comment:
let's use our original instruction for this since this isn't a class in our API 

FeatureLayer.setDefinitionExpression("Expression")

Reply:
Yes, I've reverted this